cypress/support/utils.js

```diff
@@ -170,6 +171,17 @@ export const addNewLabelToGroup = (groupName, labelName, condition, rank) => {
 
 // For ungrouped label, specify "Ungrouped" in the groupName argument
 export const labelDelete = (groupName, labelName) => {
+  cy.log("*** labelDelete (begin): groupName:labelName", groupName,labelName);
```

Comment:
nit: do we want to keep this log?

Reply:
will keep it for now while we are stabilizing things (easier to read cypress output)